lib/thesis/view.ex

```diff
@@ -193,16 +193,21 @@ defmodule Thesis.View do
     raw """
     <script>
       ;(function () {
-        var loadThesis = function () {
+        var loadThesis = function (callback) {
```

Comment:
Not @GantMan (I just play him on television).  It's best to target es5 for compat, however, [most](http://caniuse.com/#feat=const) support it.

I think the argument also for `var` here is, hoist-free shenanigans.  That script is appending crap into the `<head>` on the honour system.  Sparta.